### DISCUSSION TITLE
latin1 or similar character set

### DISCUSSION THREAD

**Discussion by mmbarret** [taviso/123elf#126] [General]
Hello, first of all I’d like to thank @taviso for his amazing work bringing lotus 1-2-3 back to life in linux, I’m totally amazed with this project.
The think is that I’m trying to get accent characters like á etc to work by configuring the  l_latin1.bun or any other character set but it does not work. By checking the issues I’ve seen that some work has been done to get UTF-8 or equivalent to work, but I don’t know how to configure it.
So is there a procedure to get this working that I’m not finding? Is there anything I can do to make it happen if that’s not the case?
Thank you very much.

**Comment by taviso** [comment]
Hey! No, I got input working, but display isn't working yet - sorry! The characters are being stored correctly internally so you're not losing any data - but I haven't got around to getting UTF-8 output working properly.

This is a good reminder I need to fix it :)

**Reply by mmbarret** [reply]
Hi! Thanks for the update! Thats great! then which character set should I configure? l_ascii or l_latin1? Thank you very much.

**Comment by taviso** [comment]
I think stay with l_ascii, because eventually utf-8 will "just work" whichever set you choose!

**Reply by mmbarret** [reply]
wow that would be nice thank you very much for your time.
